rollup.config.js

```diff
@@ -2,25 +2,68 @@ import { terser } from 'rollup-plugin-terser'
 import pkg from './package.json'
 import { nodeResolve } from '@rollup/plugin-node-resolve'
 import commonjs from '@rollup/plugin-commonjs'
+import { babel } from '@rollup/plugin-babel'
 
-const umd = { format: 'umd', name: 'A11yDialog', exports: 'default' }
-const es = { format: 'es' }
-const minify = {
-  plugins: [terser()],
-  banner: () => `/*! a11y-dialog ${pkg.version} — © Kitty Giraudel */`,
+const babelCfg = {
```

Comment:
I need to spend a bit of time digesting this config. It’s been a while I haven’t played with Babel. :)